src/scripts/core/visualization-container.js

```diff
@@ -74,6 +74,12 @@
             return this.ctx;
         },
 
+        setVisibility: function (visible) {
+            var node = this.layer.node();
+
+            visible ? $(node).show() : $(node).hide();
```

Comment:
jquery is definitely not a contour dependency - only d3 and lodash are. It's only included in the jsfiddles for the domready hook, and even that may be un-necessary.